flang/lib/Optimizer/Transforms/DebugTypeGenerator.cpp

```diff
@@ -139,10 +139,18 @@ mlir::LLVM::DITypeAttr DebugTypeGenerator::convertBoxedSequenceType(
         mlir::LLVM::DIExpressionAttr::get(context, ops);
     ops.clear();
 
+    addOp(llvm::dwarf::DW_OP_push_object_address, {});
+    addOp(llvm::dwarf::DW_OP_plus_uconst,
+          {offset + (indexSize * kDimStridePos)});
+    addOp(llvm::dwarf::DW_OP_deref, {});
+    // stride[i] = *(base_addr + offset + (indexSize * kDimStridePos))
```

Comment:
was this commented-out code left intentionally?

Reply:
No, this is just explaining how dwarf expression calculates the stride information. Similar comments exist a few line above for lower bound and count.